CMakeLists.txt

```diff
@@ -105,16 +105,15 @@ if (MILVUS_SUPPORT_SPTAG)
 endif ()
 
 if (MACOS)
-    add_subdirectory(thirdparty)
     add_subdirectory(selfdev/bruteforce)
 endif ()
 
 if (LINUX)
     include(ThirdPartyPackagesCore)
-    add_subdirectory(thirdparty)
+    add_subdirectory(knowhere/knowhere/utils)
```

Comment:
Can we compile utils like nsg? as sub dir of knowhere to execute add_subdirectory is weird?

Reply:
utils is not like nsg.
Because knowhere depends on faiss, and faiss depends on utils, we MUST build utils firstly, then faiss, then knowhere.